RaspiXUD.md

```diff
@@ -0,0 +1,183 @@
+5. Follow the inital setup instructions. Default user + password is `ubuntu`. You will be asked to change the password on first login.
+6. Update ubuntu via `sudo apt update && sudo apt upgrade`
+7. Install docker following the [official instructions](https://docs.docker.com/install/linux/docker-ce/ubuntu/) (select `arm64` in step 4 of "Set up the repository"). At the time of writing, ubuntu 19.10 was still not supported in the official docker repos. If you see an error along the lines `'docker-ce' has no installation candidate`, run this to set up the repository instead:
+```
+sudo add-apt-repository \
+   "deb [arch=arm64] https://download.docker.com/linux/ubuntu \
+   disco \
+   stable"
+```
+8. Add new user `xud`:
+
+```bash
+ubuntu@ubuntu:~$ sudo adduser xud
+Adding user `xud' ...
+Adding new group `xud' (1001) ...
+Adding new user `xud' (1001) with group `xud' ...
+Creating home directory `/home/xud' ...
+Copying files from `/etc/skel' ...
+New password: 
+Retype new password: 
+passwd: password updated successfully
+Changing the user information for xud
+Enter the new value, or press ENTER for the default
+	Full Name []: 
+	Room Number []: 
+	Work Phone []: 
+	Home Phone []: 
+	Other []: 
+
+Is the information correct? [Y/n] ubuntu@ubuntu:~$ Y
+```
+9. Add the `xud` user to the sudo group (advanced users can skip this and use another user to run sudo commands), the docker group and test if docker is working:
+```bash
+ubuntu@ubuntu:~$ sudo usermod -aG sudo xud
+ubuntu@ubuntu:~$ sudo usermod -aG docker xud
+# switch to user xud
+ubuntu@ubuntu:~$ sudo su - xud
+xud@ubuntu:~$ docker run hello-world
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+```
+10. Looking good! Optionally, add an alias to enter your xud environment by simply typing "xud":
+```bash
+xud@ubuntu:~$ sudo nano ~/.bash_aliases
+# add the line
+alias xud='bash ~/xud.sh'
+# CTRL+S, CTRL+X. Then run
+xud@ubuntu:~$ source ~/.bashrc
+```
+11. Connect an ext4-formatted USB stick to your Pi and set it up. It is very important to do this (given you do not want to loose money)!
+```bash
+# check the USB stick's path with
+xud@ubuntu:~$ ls -la /dev/ | grep sd
+crw-------  1 root root      2,  61 Dec  3 16:27 ptysd
+brw-rw----  1 root disk      8,   0 Dec  3 16:27 sda
+brw-rw----  1 root disk      8,   1 Dec  3 16:27 sda1 #this is your USB Stick
+crw-------  1 root root      3,  61 Dec  3 16:27 ttysd
+# set it to automount via fstab
+xud@ubuntu:~$ sudo nano /etc/fstab
+# add the line
+/dev/sda1 /media/USB ext4 defaults 0 2
+# CTRL+S, CTRL+X. Then mount it
+xud@ubuntu:~$ sudo mkdir /media/USB
+xud@ubuntu:~$ sudo mount -a
+# check if mounting worked
+xud@ubuntu:~$ df -h
+# make sure xud can use it
+xud@ubuntu:~$ sudo chown xud:xud /media/USB
+```
+12. Light setup - **DONE!** Continue [here](Market%20Maker%20Guide.md#the-setup).
+
+## Pi Full Setup
+
+13. Connect the ext4-formatted SSD (one large partition) to your Pi4 and set it up:
```

Comment:
Same as before. E.g. I'd like to use btrfs for my HDD and SSD. This is a filesystem which gonna be widely used in future Linux to replace ext4.

Reply:
Fixed.